packages/applications/document-builder/src/modèleRéponseSignée/index.ts

```diff
@@ -0,0 +1,6 @@
+export * from './générerModèleRéponseSignée';
+export * from './modèleRéponseSignée';
+export * from './abandon/modèleRéponseSignéeAbandon';
+export * from './garantiesFinancières/modèleRéponseSignéeMainlevée';
+export * from './garantiesFinancières/modèleRéponseSignéeMiseEnDemeure';
+export * from './recours/modèleRéponseSignéeRecours';
```

Comment:
a priori pas d'intéret d'exporter tout ca ? seul `générerModèleRéponseSignée` est utilisé ?
```suggestion
export * from './générerModèleRéponseSignée';
```